news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md

```diff
@@ -0,0 +1,210 @@
+
+* [#359](https://github.com/PrestaShop/docs/pull/359): Add Laragon as an WAMP environment. Thank you [@yannicka](https://github.com/yannicka)
+
+### blockreassurance
+
+* [#32](https://github.com/PrestaShop/blockreassurance/pull/32): Release 4.1.1, by [@Progi1984](https://github.com/Progi1984)
+
+* [#31](https://github.com/PrestaShop/blockreassurance/pull/31): Release 4.1.1, by [@Progi1984](https://github.com/Progi1984)
+
+* [#30](https://github.com/PrestaShop/blockreassurance/pull/30): Fixed autoload on PS 1.7.5.x (and before), by [@Progi1984](https://github.com/Progi1984)
+
+* [#29](https://github.com/PrestaShop/blockreassurance/pull/29): Release 4.1.0, by [@Progi1984](https://github.com/Progi1984)
+
+* [#28](https://github.com/PrestaShop/blockreassurance/pull/28): Release 4.1.0, by [@Progi1984](https://github.com/Progi1984)
+
+* [#27](https://github.com/PrestaShop/blockreassurance/pull/27): Fix feedback (Add block, design, responsive), by [@Progi1984](https://github.com/Progi1984)
+
+### core-weekly-generator
+
+* [#6](https://github.com/PrestaShop/core-weekly-generator/pull/6): Fix format error, by [@matks](https://github.com/matks)
+
+* [#5](https://github.com/PrestaShop/core-weekly-generator/pull/5): Refactoring, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+* [#4](https://github.com/PrestaShop/core-weekly-generator/pull/4): Small fix about blank line at the top, by [@matks](https://github.com/matks)
+
+* [#3](https://github.com/PrestaShop/core-weekly-generator/pull/3): V2 with python, by [@matks](https://github.com/matks)
+
+### ps_emailsubscription
+
+* [#44](https://github.com/PrestaShop/ps_emailsubscription/pull/44): Release 2.5.1, by [@matks](https://github.com/matks)
+
+* [#43](https://github.com/PrestaShop/ps_emailsubscription/pull/43): Bump to v2.5.1, add contributors file, by [@matks](https://github.com/matks)
+
+* [#41](https://github.com/PrestaShop/ps_emailsubscription/pull/41): Fix compatibility of v2.5 with PS 1.7.6 and 1.7.5, by [@matks](https://github.com/matks)
+
+### TranslationTools Bundle
+
+* [#67](https://github.com/PrestaShop/TranslationToolsBundle/pull/67): Allow twig 1 instead only twig 2, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+### CsaGuzzleBundle
+
+* [#6](https://github.com/PrestaShop/CsaGuzzleBundle/pull/6): Allow twig 1 instead only twig 2, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+* [#5](https://github.com/PrestaShop/CsaGuzzleBundle/pull/5): Update README of master branch with fork message, by [@matks](https://github.com/matks)
+
+### Classic-rocket theme
+
+* [#100](https://github.com/PrestaShop/classic-rocket/pull/100): Change type for phone_mobile field. Thank you [@micka-fdz](https://github.com/micka-fdz)
+
+### productcomments
+
+* [#38](https://github.com/PrestaShop/productcomments/pull/38): Fix for product display in modal. Thank you [@Oksydan](https://github.com/Oksydan)
+
+### PrestaShop-1.6
```

Comment:
We might need to improve this title in order for people not confuse it with main repository

Reply:
yes. maybe adding legacy or something like that. BTW, maybe we should update the name of the repository or the repository title.